api/agentpb/agent.proto

```diff
@@ -139,24 +147,48 @@ message StartActionRequest {
     string dsn = 1;
     string query = 2;
     MysqlExplainOutputFormat output_format = 3;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles tls_files = 4;
+    // TLS in enabled if true.
+    bool tls = 6;
```

Comment:
maybe non-empty tls_files should be enough for check, what do you think?

Reply:
Good idea. Changed to check textfile instead.